src/components/pages/Profile/RenderProfileContainer.js

```diff
@@ -11,13 +11,14 @@ import {
   GithubOutlined,
 } from '@ant-design/icons';
 
-import axiosWithAuth from '../../../utils/axiosWithAuth';
+import useAxiosWithAuth0 from '../../../hooks/useAxiosWithAuth0';
 
 const { Title } = Typography;
 const { Meta } = Card;
 
 const RenderProfileContainer = props => {
   const [userData, setUserData] = useState({});
+  const { axiosWithAuth } = useAxiosWithAuth0();
```

Comment:
Why would this step be necessary instead of just using useAxiosWithAuth0 directly?  Same with using it inside of useEffect below

Reply:
Well, it's exported from that file as `axiosWithAuth`, which is a standard for making use of a custom `axios` instance that makes use of some 3rd party authentication software - this also helps devs on this project who've been around for some time know to continue using the same function name they've been using right along, which allows for continued ease-of-use. 🙂